NuCivic/dkan
# Fixed count of overridden nodes in harvest

Issue: https://jira.govdelivery.com/browse/CIVIC-5124

## Description

When you use filter on harvest nodes is showing the same number for overriden and filter.

## How to reproduce

1.  Create a harvest source (I used http://www.usda.gov/data.json )
2. Add a filter ( keyword : ARS )
3. Click save to cache the data
4. You will see the number of overridden datasets equals the number of filtered datasets

## QA Tests

- [x]  Create a new harvest node with filter (For example from the steps to reproduce)
- [x] You should be only number in filter message and 0 for overriden
